includes/Customizer.php

```diff
@@ -320,67 +343,68 @@ public function register_customizer_settings( \WP_Customize_Manager $wp_customiz
 			);
 		}
 
-		if ( true === $theme_support['stories-archive-link'] ) {
+		if ( true === $theme_support['sharp-corners'] ) {
 			$wp_customize->add_setting(
-				self::STORY_OPTION . '[show_stories_archive_link]',
+				self::STORY_OPTION . '[sharp_corners]',
 				[
-					'default' => false,
+					'default' => $theme_support['sharp-corners'],
 					'type'    => 'option',
 				]
 			);
 
 			$wp_customize->add_control(
-				self::STORY_OPTION . '[show_stories_archive_link]',
+				self::STORY_OPTION . '[sharp_corners]',
 				[
 					'type'            => 'checkbox',
 					'section'         => self::SECTION_SLUG,
-					'label'           => __( 'Show stories archive link', 'web-stories' ),
+					'label'           => __( 'Sharp Corners', 'web-stories' ),
 					'active_callback' => function() {
-						return $this->is_option_enabled( 'show_stories' );
+						return ( $this->is_option_enabled( 'show_stories' ) && ! $this->is_view_type( 'circles' ) );
 					},
 				]
 			);
+		}
 
+		if ( true === $theme_support['stories-archive-link'] ) {
 			$wp_customize->add_setting(
-				self::STORY_OPTION . '[stories_archive_label]',
+				self::STORY_OPTION . '[show_stories_archive_link]',
 				[
+					'default' => false,
 					'type'    => 'option',
-					'default' => $theme_support['stories-archive-label'],
 				]
 			);
 
 			$wp_customize->add_control(
-				self::STORY_OPTION . '[stories_archive_label]',
+				self::STORY_OPTION . '[show_stories_archive_link]',
 				[
-					'type'            => 'text',
+					'type'            => 'checkbox',
 					'section'         => self::SECTION_SLUG,
-					'label'           => __( 'Stories archive label', 'web-stories' ),
+					'label'           => __( 'Show stories archive link', 'web-stories' ),
 					'active_callback' => function() {
-						return ( $this->is_option_enabled( 'show_stories' ) && $this->is_option_enabled( 'show_stories_archive_link' ) );
+						return $this->is_option_enabled( 'show_stories' );
 					},
 				]
 			);
 
 			$wp_customize->add_setting(
-				self::STORY_OPTION . '[show_story_poster]',
+				self::STORY_OPTION . '[stories_archive_label]',
 				[
-					'default' => $theme_support['show-story-poster-default'],
 					'type'    => 'option',
+					'default' => $theme_support['stories-archive-label'],
 				]
 			);
 
 			$wp_customize->add_control(
-				self::STORY_OPTION . '[show_story_poster]',
+				self::STORY_OPTION . '[stories_archive_label]',
 				[
-					'type'            => 'checkbox',
+					'type'            => 'text',
 					'section'         => self::SECTION_SLUG,
-					'label'           => __( 'Show story poster', 'web-stories' ),
+					'label'           => __( 'Stories archive label', 'web-stories' ),
```

Comment:
```suggestion
					'label'           => __( 'Archive Link Label', 'web-stories' ),
```

That's what we use in the Customizer I think?

Let's make sure we use the same labels everywhere.